SCClassLibrary/Common/Quarks/Git.sc

```diff
@@ -48,6 +48,22 @@ Git {
 		});
 		^nil
 	}
+	httpsString {
```

Comment:
as telephon pointed out, it is not necessarily a github url. also, 20 chars... also, `get` is (fortunately) rarely used in sclang. 

+ `remoteAsHttps`?